app/src/main/res/layout/view_ssl_warning.xml

```diff
@@ -0,0 +1,150 @@
+
+        <com.duckduckgo.common.ui.view.button.DaxButtonGhost
+            android:id="@+id/sslErrorAdvancedCTA"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginHorizontal="@dimen/keyline_5"
+            android:text="@string/sslErrorAdvancedCTA"
+            app:buttonSize="large"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/sslErrorLeaveSiteCTA"/>
+
+        <androidx.constraintlayout.widget.Group
+            android:id="@+id/sslErrorAdvancedGroup"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:visibility="gone"
+            app:constraint_referenced_ids="sslErrorAdvancedDivider, sslErrorExpandedHeadline, sslErrorExpandedMessage, sslErrorAcceptCta"  />
+
+        <com.duckduckgo.common.ui.view.divider.HorizontalDivider
+            android:id="@+id/sslErrorAdvancedDivider"
+            app:defaultPadding="false"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/sslErrorAdvancedCTA"
+            android:layout_marginTop="@dimen/keyline_5"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"/>
+
+        <com.duckduckgo.common.ui.view.text.DaxTextView
+            android:id="@+id/sslErrorExpandedHeadline"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/keyline_5"
+            android:layout_marginHorizontal="@dimen/keyline_5"
+            android:gravity="center"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/sslErrorAdvancedDivider"
+            app:typography="body2"
+            android:textAlignment="center"
+            tools:text="@string/sslErrorExpandedHeadline" />
+
+        <com.duckduckgo.common.ui.view.text.DaxTextView
+            android:id="@+id/sslErrorExpandedMessage"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/keyline_4"
+            android:layout_marginHorizontal="@dimen/keyline_5"
+            android:gravity="center"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/sslErrorExpandedHeadline"
+            app:typography="body2"
+            android:textAlignment="center"
+            tools:text="@string/sslErrorExpiredMessage" />
+
+        <com.duckduckgo.common.ui.view.text.DaxTextView
+            android:id="@+id/sslErrorAcceptCta"
```

Comment:
This view is too close to the bottom of the screen on landscape or a small screen where you need to scroll and it's hard to tap on it.